package.json

```diff
@@ -105,6 +105,7 @@
     "jest-cli": "^25.1.0",
     "jest-styled-components": "^7.0.0",
     "jsdom": "^15.1.1",
+    "mnet-icons": "git+ssh://git@tree.mn:saumik.s/mnet-icons.git",
```

Comment:
this we can't have here. you can do it for local development but not here

Reply:
mnet-icons dependency is now removed from devDependencies array